weave-js/src/components/Panel2/PanelGroup.tsx

```diff
@@ -17,6 +31,7 @@ import {Draft, produce} from 'immer';
 import * as _ from 'lodash';
 import React, {useCallback, useMemo} from 'react';
 import {Button} from 'semantic-ui-react';
+import {Button as NewButton} from '../Button';
```

Comment:
I'd have a slight preference for importing the Semantic UI button as OldButton instead. Would be interested in what's keeping us from replacing it with a new Button, i.e. is the styling such that it doesn't map onto any of our new button variants?

Reply:
I changed flipped it to OldButton, Button. I'm sure work will come along soon where we want to update the places in this file that OldButton is used, we can fix it then.